app/src/lib/ui/Manager/ImportBeneficiaries.svelte

```diff
@@ -147,18 +162,18 @@
 	async function handleFilesSelect(event: CustomEvent<{ acceptedFiles: Buffer[] }>): Promise<void> {
 		parseErrors = [];
 		files = event.detail.acceptedFiles;
+		parsing = true;
 		for (let i = 0; i < files.length; i++) {
-			const parsingResponse = await fileValidationApi(files[i]);
-			const responseBody = await parsingResponse.json();
-			if (parsingResponse.ok) {
-				beneficiaries = responseBody.map((line) => beneficiaryFrom(line));
+			try {
+				beneficiaries = await fileValidationApi(files[i]);
+
 				toImport = beneficiaries
 					.filter((beneficiary) => beneficiary.valid)
 					.map((beneficiary) => beneficiary.uid);
-			} else if (responseBody.detail) {
-				parseErrors.push(responseBody.detail);
-			} else {
-				parseErrors.push('Unknown error while parsing imported file');
+			} catch (error) {
+				parseErrors.push(error);
+			} finally {
+				parsing = false;
```

Comment:
si le booléen `parsing` sert à indiquer que l'ensemble des fichiers est en cours de `parsing` alors il faut le mettre à `false` après la boucle `for`